webpack.config.js

```diff
@@ -0,0 +1,38 @@
+const path = require("path");
```

Comment:
Similar to `.eslintrc.js`, have you looked at the webpack config in existing projects? It would be good to keep these configurations as consistent as possible.

Reply:
This one is taken from https://github.com/ubuntudesign/360.canonical.com/blob/master/webpack.config.js .

Similar to https://github.com/canonical-websites/snapcraft.io/blob/master/webpack.config.js, except for the modification possibilities at the bottom of the config. 

Snapcraft also has some module exposing which is unnecessary here.